django-backend/fecfiler/transactions/views.py

```diff
@@ -149,10 +149,12 @@ def update(self, request, *args, **kwargs):
             saved_transaction = self.save_transaction(request.data, request)
         return Response(saved_transaction.id)
 
-    def delete(self, request, *args, **kwargs):
-        response = super().delete(request, *args, **kwargs)
-        update_dependent_parent(self.get_object())
-        return response
+    def destroy(self, request, *args, **kwargs):
+        pk = kwargs.get("pk")
+        with db_transaction.atomic():
+            instance = Transaction.objects.get(pk=pk)
```

Comment:
In general we want to avoid using the base querysets in our views like `Transaction.objects.` and use the class's `get_queryset()` to preserve the access controls we have.   As this is written anyone could delete anyone's transaction, but if we use `get_queryset()` or `self.get_object()` (which depends on `get_queryset()`) we can limit access to the users' transactions.

We should have added tests for that so it would have been flagged after this change.  Maybe we can add some in.